lib/solidus_support/engine_extensions/decorators.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SolidusSupport
+  module EngineExtensions
+    module Decorators
+      def self.included(base)
+        base.config.to_prepare(&method(:activate).to_proc)
+      end
+
+      def activate
+        base_path = File.join(root, "app/decorators")
```

Comment:
`root` is already a Pathname that implements `.join`

Reply:
I like the idea of a dedicated folder, but doesn't this break existing engines, that uses the old `*_decorator` naming?

Reply:
Yes it will break them, but you are free to don't use this module if you are still stick to the old way. Also, moving things around is required to make decorators work with Rails 6 and Zeitwerk. The idea is to make these two things together in extensions.

Reply:
I see. Makes sense.

Reply:
for `.root`, you are probably right. I thought `.root` was only available for the main Rails application but it's probably also available for Engine as well. Let me try!